composer/algorithms/stochastic_depth/stochastic_depth.py

```diff
@@ -38,7 +38,8 @@ def apply_stochastic_depth(model: torch.nn.Module,
                            drop_distribution: str = 'linear',
                            use_same_gpu_seed: bool = True,
                            optimizers: Optional[Union[Optimizer, Sequence[Optimizer]]] = None) -> torch.nn.Module:
-    """Applies Stochastic Depth (`Huang et al, 2016 <https://arxiv.org/abs/1603.09382>`_) to the specified model.
+    """Applies Stochastic Depth (`Huang et al, 2016 <https://arxiv.org/abs/1603.09382>`_) 
```

Comment:
File needs a module-level docstring and `__all__`.

Reply:
Added now.